### PR TITLE
CEDS-1814 Arrival moment validation

### PR DESCRIPTION
**Pull request by bambuchaAdm**

### CODE REVIEW

File: app/forms/ArrivalDetails.scala [hunk 1]
```diff
@@ -16,17 +16,23 @@
...
-import java.time.ZoneId
+import java.time.{ZoneId, ZonedDateTime}
 
 import forms.common.{Date, Time}
 import play.api.data.{Forms, Mapping}
 import play.api.libs.json.{Json, OFormat}
 
-case class ArrivalDetails(dateOfArrival: Date, timeOfArrival: Time)
+case class ArrivalDetails(dateOfArrival: Date, timeOfArrival: Time) {
+  def goodsArrivalMoment(zoneId: ZoneId): ZonedDateTime =
```
Comment (PatrykRudnicki) [inline app/forms/ArrivalDetails.scala]: I'll change the name to something different than Moment. It's not enough explanatory for me.

File: test/forms/ArrivalDetailsSpec.scala [hunk 2]
```diff
@@ -48,6 +53,18 @@ class ArrivalDetailsSpec extends BaseSpec {
 
         errors.length must be(2)
       }
+
+      "moment is in future" in {
```
Comment (PatrykRudnicki) [inline test/forms/ArrivalDetailsSpec.scala]: I'll change moment to something different

**Comment by PatrykRudnicki** [comment]
Please use scalafmt

File: test/forms/ArrivalDetailsSpec.scala [hunk 3]
```diff
@@ -48,6 +53,18 @@ class ArrivalDetailsSpec extends BaseSpec {
...
+        val form  = movementDetails.arrivalForm().bind(dateMapping.unbind(Date(LocalDate.now().plusDays(1))) ++ timeInputData)
+
+        form.errors.flatMap(_.messages) must contain("arrival.details.error.future")
+      }
+
+      "moment is is more then 60 days in past" in {
```
Comment (MaciejRewera) [inline test/forms/ArrivalDetailsSpec.scala]: Nit: There is a typo: double 'is'

**Pull request merged**